index.html

```diff
@@ -58,6 +57,158 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
 				</ul>
 			</section>
 		</section>
+		<section class="works-section">
+			<ul class="works">
```

Comment:
- [ ] It looks you're using 8 spaces for indentation in all the files, please, use only 2 spaces. 🐶 
![image](https://user-images.githubusercontent.com/20563252/125117879-77963580-e0b4-11eb-8f6f-205a8d0b745a.png)